src/foam/u2/tag/Select.js

```diff
@@ -97,10 +104,12 @@ foam.CLASS({
         for ( var i = 0 ; i < choices.length ; i++ ) {
           var c = choices[i];
           let value = c[1];
+          var isDisabled = self.slot(function(data) { return self.isOptionDisabled_(value, data === i); });
```

Comment:
wonder if isDisabled need to be a slot?

Reply:
so the problem this solves is that without a slot the currently selected item cannot be detected by `data === i` as ChoiceView.index does not seem to be set before Select.render call.
this was the solution Sarthak and I came up with to allow the currently selected item to not be disabled so you can select it again to close the select

Reply:
Removed use of slot for data